src/Tasks/Resources/Strings.resx

```diff
@@ -1161,6 +1161,11 @@
     <value>MSB3824: In order to build with .NET Core, resource inputs must be in .txt or .resx format.</value>
     <comment>{StrBegin="MSB3824: "}</comment>
   </data>
+  <data name="GenerateResource.BinaryFormatterUse">
+    <value>MSB3825: Resource "{0}" of type "{2}" is deserialized via BinaryFormatter at runtime. BinaryFormatter is deprecated due to possible security risks and will be removed with .NET 9. If you wish to continue using it, set property "GenerateResourceWarnOnBinaryFormatterUse" to false.
+           More information: https://learn.microsoft.com/dotnet/standard/serialization/binaryformatter-security-guide</value>
```

Comment:
Isn't there a helplink field on the warning? can we set that too?

Reply:
This sounds familiar, but I'm having trouble finding what I'd expected.

Searching for "helplink" didn't return anything helpful.
Searching for "help" in the TaskLoggingHelper revealed that there's a HelpKeywordPrefix (generally just set to MSBuild) that's added to the messageResourceName, so no action there, but I don't think that's what you meant.
Searching for "help" in the same Strings.resx revealed nothing.
Searching the whole repo for <help also didn't work.

Any suggestions?

Reply:
We noticed that HelpLink is available, but there isn't a helper method for it yet. I suggested I'd make a follow-up PR to fix that and clean up all the other instances using it.

Reply:
https://github.com/dotnet/msbuild/issues/5493#issuecomment-1238926812 <-- we don't actually have an easy way to do this right now. So we can wait.